src/draw/dma2d/lv_draw_dma2d_private.h

```diff
@@ -19,7 +19,14 @@ extern "C" {
 
 #include "../lv_draw_private.h"
 #include "../sw/lv_draw_sw.h"
+#if defined(__ZEPHYR__)
```

Comment:
Wouldn't it be more consistent to have a header within zepyhr gluecode and set LV_DRAW_DMA2D_HAL_INCLUDE ?

Reply:
Yup, all good.